requirements/environment.yml

```
@@ -5,225 +5,336 @@ channels:
+  - libopenblas=0.3.24
   - libopus=1.3.1
-  - libpng=1.6.37
-  - libpq=13.3
-  - libssh2=1.10.0
-  - libstdcxx-ng=11.2.0
-  - libtiff=4.3.0
-  - libuuid=2.32.1
+  - libpng=1.6.39
+  - libpq=16.1
+  - libsndfile=1.2.2
+  - libsolv=0.7.26
+  - libsqlite=3.44.0
+  - libssh2=1.11.0
+  - libstdcxx-ng=13.2.0
+  - libsystemd0=254
+  - libtiff=4.6.0
+  - libuuid=2.38.1
   - libvorbis=1.3.7
-  - libwebp=1.2.1
-  - libwebp-base=1.2.1
-  - libxcb=1.13
-  - libxkbcommon=1.0.3
-  - libxml2=2.9.12
-  - libzip=1.8.0
-  - libzlib=1.2.11
+  - libwebp-base=1.3.2
+  - libxcb=1.15
+  - libxkbcommon=1.6.0
+  - libxml2=2.11.5
+  - libzip=1.10.1
+  - libzlib=1.2.13
   - livereload=2.6.3
-  - llvm-openmp=12.0.1
-  - lz4-c=1.9.3
-  - markupsafe=2.0.1
-  - matplotlib=3.4.3
-  - matplotlib-base=3.4.3
-  - matplotlib-inline=0.1.3
-  - mccabe=0.6.1
-  - more-itertools=8.11.0
+  - lz4-c=1.9.4
+  - lzo=2.10
+  - mamba=1.4.2
+  - markdown-it-py=3.0.0
+  - markupsafe=2.1.3
+  - matplotlib=3.8.1
+  - matplotlib-base=3.8.1
+  - matplotlib-inline=0.1.6
+  - mccabe=0.7.0
+  - mdurl=0.1.0
+  - more-itertools=10.1.0
+  - mpg123=1.32.3
   - munkres=1.1.4
-  - mypy=0.910
-  - mypy_extensions=0.4.3
-  - mysql-common=8.0.27
-  - mysql-libs=8.0.27
-  - ncurses=6.2
-  - netcdf4=1.5.8
-  - nodeenv=1.6.0
-  - nspr=4.32
-  - nss=3.72
-  - numpy=1.22.3
-  - olefile=0.46
-  - openjpeg=2.4.0
-  - openssl=1.1.1l
-  - packaging=21.0
-  - parso=0.8.2
-  - pathspec=0.9.0
-  - pcre=8.45
+  - mypy=1.7.0
+  - mypy_extensions=1.0.0
+  - mysql-common=8.0.33
+  - mysql-libs=8.0.33
+  - nbformat=5.9.2
+  - ncurses=6.4
+  - netcdf4=1.6.5
+  - nodeenv=1.8.0
+  - nspr=4.35
+  - nss=3.94
+  - numpy=1.26.0
+  - openjpeg=2.5.0
+  - openssl=3.1.4
+  - packaging=23.2
+  - parso=0.8.3
+  - patch=2.7.6
+  - patchelf=0.17.2
+  - pathspec=0.11.2
+  - pcre2=10.42
   - pexpect=4.8.0
   - pickleshare=0.7.5
-  - pillow=8.4.0
-  - pip=21.3.1
-  - platformdirs=2.3.0
-  - pluggy=0.13.1
-  - pre-commit=2.16.0
-  - proj=8.2.0
-  - prompt-toolkit=3.0.22
-  - psutil=5.8.0
+  - pillow=10.1.0
+  - pip=23.3.1
+  - pixman=0.42.2
+  - pkginfo=1.9.6
+  - pkgutil-resolve-name=1.3.10
+  - platformdirs=3.11.0
+  - pluggy=1.3.0
+  - ply=3.11
+  - pre-commit=3.5.0
+  - proj=9.3.0
+  - prompt_toolkit=3.0.41
+  - psutil=5.9.5
   - pthread-stubs=0.4
   - ptyprocess=0.7.0
-  - py=1.11.0
-  - pycodestyle=2.8.0
+  - pulseaudio-client=16.1
+  - pure_eval=0.2.2
+  - py-lief=0.12.3
+  - pybind11-abi=4
+  - pycodestyle=2.11.1
+  - pycosat=0.6.6
   - pycparser=2.21
-  - pydocstyle=6.1.1
-  - pyflakes=2.4.0
-  - pygments=2.10.0
-  - pylint=2.11.1
-  - pyopenssl=21.0.0
-  - pyparsing=3.0.6
-  - pypdf2=1.26.0
-  - pyproj=3.2.1
-  - pyqt=5.12.3
-  - pyqt-impl=5.12.3
-  - pyqt5-sip=4.19.18
-  - pyqtchart=5.12
-  - pyqtwebengine=5.12.1
-  - pyshp=2.1.3
+  - pydantic=2.5.1
+  - pydantic-core=2.14.3
+  - pydocstyle=6.3.0
+  - pyflakes=3.1.0
+  - pygments=2.16.1
+  - pylint=3.0.2
+  - pyopenssl=23.3.0
+  - pyparsing=3.1.1
+  - pypdf2=1.28.4
```

```suggestion
  - pypdf=3.10.0
```
This fixes the issue with the ImportError

---

Works fine also on balfrin.